mlflow/protos/service.proto

```diff
@@ -75,6 +75,22 @@ service MlflowService {
     };
   }
 
+  rpc searchExperiments (SearchExperiments) returns (SearchExperiments.Response) {
+    option (rpc) = {
+      endpoints: [{
+        method: "POST",
+        path: "/mlflow/experiments/search"
+        since { major: 2, minor: 0 },
+      }, {
+        method: "POST",
+        path: "/preview/mlflow/experiments/search"
+        since { major: 2, minor: 0 },
+      }],
```

Comment:
No need for a preview API here. We'll be dropping all preview routes from MLflow in 2.0 & don't need to include preview routes for new APIs.

Reply:
Got it. I'll remove it.